benchmark.py

```diff
@@ -0,0 +1,38 @@
+import scmdata
```

Comment:
@mikapfl Did you mean to add this file?

Reply:
nope, sorry for the confusion. I thought I deleted it, but apparently I added it to the commit instead, lol.